lib/services/local/presence.go

```diff
@@ -1559,7 +1559,7 @@ func (s *PresenceService) DeleteAllWindowsDesktopServices(ctx context.Context) e
 func (s *PresenceService) ListResources(ctx context.Context, req proto.ListResourcesRequest) (*types.ListResourcesResponse, error) {
 	switch {
 	case req.RequiresFakePagination():
-		return s.listResourcesWithSort(ctx, req)
+		return nil, trace.BadParameter("request requiring getXXX resource methods are not supported in presence.ListResources")
```

Comment:
i didn't think this path will ever be used (bypassing auth with roles layer), so i thought it would be nice to have one less thing to maintain, but i can see how it can cause confusion. i reverted it back and also placed de-duplication and tests